docs/reference/rest-api/common-parms.asciidoc

```diff
@@ -999,15 +999,17 @@ end::transform-metadata[]
 
 tag::transform-retention[]
 Defines a retention policy for the {transform}. Data that meets the defined
-criteria is deleted from the destination index.
+criteria is deleted from the destination index. Data is deleted if `time.field` 
+for the retention policy exists and contains data older than `max.age`.
 end::transform-retention[]
```

Comment:
I think the new sentence should be placed one section below in `tag::transform-retention-time[]`.

(Although we haven't implemented one yet, the API allows _other_ retention policies.)

Reply:
I moved the sentence under `transform-retention-time` via [4fb50a5](https://github.com/elastic/elasticsearch/pull/89685/commits/4fb50a5fc28471c9ca6125a7b5612e09c8f94d4f).